<?xml version="1.0"?>
<launch>
    
  <arg name="tool_name" default="fts"/>
  <arg name="store_to_file" default="true"/>
  <arg name="robot_name" default="r5"/>
  <arg name="robot" default="schunk"/>
  
  <node pkg="force_torque_sensor" type="calibrate_tool.py" name="calibrate_tool" cwd="node" respawn="false" output="screen">
      <param name="tool_name" value="$(arg tool_name)"/>
      <param name="store_to_file" value="$(arg store_to_file)"/>
      <param name="robot_name" value="$(arg robot_name)"/>
      <param name="robot" value="$(arg robot)"/>
  </node>

</launch>
